v2/CHANGES

```diff
@@ -40,6 +40,7 @@ Encodings:
 - [x] Drop raw encoding
 - [x] Port JSON encoding
 - [ ] Port Thrift encoding
+      - [ ] Revise/Add comment about final implementation in outbound#Call
```

Comment:
can we drop note and nuke the comment in `Outbound.Call`? The comment there seems unnecessary.

Reply:
I've factored this into the plugin branch, so it'll be gone when I merge all of them.